src/bundle/Resources/public/css/variables.css

```diff
@@ -0,0 +1,37 @@
+:root {
+    --ez-base-font-family: "Noto Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
You're right Noto is not a standard font. The idea behind using Noto is to support a lot of languages for our new admin interface. We were thinking to include Noto as a font in the Hybride-
Platform-UI bundle.
Regarding the fallback to Helvetica or Arial, we can keep it. +1